data-explorer/kusto/concepts/querylimits.md

```diff
@@ -81,6 +81,12 @@ set truncationmaxrecords=1105;
 MyTable | where User=="Ploni"
 ```
 
+When considering removing these limits, first determine if
```

Comment:
I suggest moving the first sentence of this paragraph to the end of the first paragraph of the article, as it relates to all the limits. The rest of the paragraph should indeed be where you planted it.

Reply:
Agree & done.